SwiftCompilerSources/Sources/Basic/Utils.swift

```diff
@@ -52,7 +52,9 @@ public extension NoReflectionChildren {
   var customMirror: Mirror { Mirror(self, children: []) }
 }
 
+#if !os(Windows)
 public var standardError = CFileStream(fp: stderr)
+#endif
```

Comment:
You should be able to implement this with the Win32 + ucrt APIs to convert the stderr handle to a `FILE *`.